src/Client/GuzzleRequestFactory.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Juicer\Client;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Utils;
+use Keboola\Juicer\Exception\UserException;
+use Psr\Http\Message\RequestInterface;
+use function Keboola\Utils\buildUrl;
+
+/**
+ * Creates GuzzleRequest from RestRequest
+ */
+class GuzzleRequestFactory
```

Comment:
Povodne to bola metoda `RestClient::getGuzzleRequest` ktoru ale neslo otestovat bez volania private metody, tak som to vyclenil, aby som si bol isty, ze som to nerozbil, je na to `GuzzleRequestFactoryTest`